<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(212, 178, 172);</string>
  </property>
  <widget class="QLineEdit" name="edt_assunto">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>90</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
padding: 10px;
border: 1px solid gray;</string>
   </property>
  </widget>
  <widget class="QLabel" name="descricao2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>60</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Assunto a inserir:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editar_assunto">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 147, 214);
color: rgb(255, 255, 255);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Editar assuntos</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>14</y>
     <width>348</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="rdb_Mat">
      <property name="text">
       <string>Matemática</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rdb_Port">
      <property name="text">
       <string>Português</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rdb_Sau">
      <property name="text">
       <string>Saúde</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rdb_facul">
      <property name="text">
       <string>Faculdade</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_inserir">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>92</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 189, 140);
color: rgb(255, 255, 255);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Inserir</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
